ddtrace/settings/integration.py

```diff
@@ -50,9 +50,16 @@ def __init__(self, global_config, name, *args, **kwargs):
             analytics_enabled_env = asbool(analytics_enabled_env)
         self.setdefault("analytics_enabled", analytics_enabled_env)
         old_analytics_rate = get_env(name, "analytics_sample_rate", default=1.0)
+
         analytics_rate = os.environ.get("DD_TRACE_%s_ANALYTICS_SAMPLE_RATE" % name.upper(), old_analytics_rate)
         self.setdefault("analytics_sample_rate", float(analytics_rate))
 
+        service = get_env(name, "service")
+        self.setdefault("service", service)
+        # TODO: this is required for backwards compatibility since some
```

Comment:
`TODO` what then?

Reply:
Yeah it's kinda a v1.0 note probably not worthy of a TODO.